doc/building_blocks/manual_pipeline.rst

```diff
@@ -135,9 +135,17 @@ The :class:`NiftiMasker` can be seen as a *tube* that transforms data
 from 4D images to 2D arrays, but first it needs to 'fit' this data in
 order to learn simple parameters from it, such as its shape:
 
-.. literalinclude:: ../../examples/plot_decoding_tutorial.py
-    :start-after: # Now we use the NiftiMasker.
-    :end-before: ###########################################################################
+.. code-block:: python
```

Comment:
Those "code-block" directives are not maintainable. We have stopped using them, and prefer to put a lot of text in the examples, and linking from the narrative docs to the example.

Maybe put this aside for a moment, as someone has just improved a lot the example in another PR ( #2260 )

Reply:
Put this aside how?
So we should merge #2260 before this one and then adapt this accordingly?